policy/predicate/file.go

```diff
@@ -64,20 +78,32 @@ func (pred *OnlyChangedFiles) Evaluate(ctx context.Context, prctx pull.Context)
 		return false, "", errors.Wrap(err, "failed to parse paths")
 	}
 
+	ignorePaths, err := pathsToRegexps(pred.IgnorePaths)
+	if err != nil {
+		return false, "", errors.Wrap(err, "failed to parse ignore paths")
+	}
+
 	files, err := prctx.ChangedFiles()
 	if err != nil {
 		return false, "", errors.Wrap(err, "failed to list changed files")
 	}
 
+	var relevantFiles []string
 	for _, f := range files {
-		if anyMatches(paths, f.Filename) {
+		if !anyMatches(ignorePaths, f.Filename) {
```

Comment:
Similar to above, I think this could be combined into one loop, but it depends on what we decide the behavior should be.